.github/workflows/createDockerContainer.yml

```diff
@@ -13,7 +13,7 @@
 
  jobs:
    build-docker-debian:
-     if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
why is this here? Some broken rebasing?

Reply:
On main this is already there i think?

Reply:
Yes this was because of a broken rebase I assume. This is not there in the `main` branch though

Reply:
This commit was dropped. It appears here due to some rebasing error I assume.